src/bundle/ui-dev/src/modules/universal-discovery/bookmarks.tab.module.js

```diff
@@ -102,8 +102,15 @@ const BookmarksTabModule = () => {
     return (
         <div className="m-bookmarks-tab">
             <Tab>
-                <BookmarksList itemsPerPage={tabsConfig.bookmarks.itemsPerPage} setBookmarkedLocationMarked={setBookmarkedLocationMarked} />
-                {isMarkedLocationSetByBookmarksRef.current && renderBrowseLocations()}
+                {restorationStateRef.current ? (
```

Comment:
`: null` won't be needed if we do:
```suggestion
                {restorationStateRef.current && (
```